org-formation/050-costs/program-code-rules.json

```diff
@@ -0,0 +1,399 @@
+[
+  {
+    "Type": "REGULAR",
+    "Value": "Platform Infrastructure",
+    "Rule": {
+      "And": [
+    {
+      "Dimensions": {
+        "Key": "LINKED_ACCOUNT",
+        "Values": [
+          "531805629419",
+          "140124849929",
+          "254940560790",
+          "231505186444",
+          "153370007719",
+          "087547153226",
+          "797640923903",
+          "867686887310",
+          "420786776710",
+          "745159704268",
+          "325565585839",
+          "383874245509",
+          "449435941126",
+          "649232250620",
+          "804034162148",
+          "743644221192"
```

Comment:
why not list all accounts here?

if you want to be able to reference specific account IDs by using something like `!Ref SandboxAccount` then i think you can do that with [orgformation templating](https://github.com/org-formation/org-formation-cli/blob/master/docs/task-files.md#example).  if you want to generate a list of all account IDs then you can use the [Fn::EnumTargetAccounts feature](https://github.com/org-formation/org-formation-cli/blob/master/docs/cloudformation-resources.md#fnenumtargetaccountsregions).

Reply:
1) This is a pure json file intended to be read in as a string in the template using `!ReadFile`. 2) These are the accounts that are tagged with "Platform Infrastructure", I was planning on leaving untagged resources in other accounts as uncategorized for now; but if we want categorize absolutely everything that isn't tagged as "Platform Infrastructure", then it would be better to remove this rule entirely and use "Platform Infrastructure" as a final default value.